library/src/main/java/com/getstream/sdk/chat/rest/core/Client.java

```diff
@@ -1088,6 +1090,57 @@ public void onFailure(Call<MessageResponse> call, Throwable t) {
         });
     }
 
+    /**
+     * list the reactions, supports pagination
+     *
+     * @param messageId the message id
+     * @param limit     pagination param
+     * @param offset    pagination param
+     * @param callback  the result callback
+     */
+    public void getReactions(@NotNull String messageId,
```

Comment:
for consistency with other clients it would be better to an option class to capture offset / limit (easier to add more options in the future as well). Can you add that?

Reply:
added PaginationOptions